docs/guides/cloud/data-storage-and-masking.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Data Storage and Controls
+sidebar_position: 60
+sidebar_label: Data Storage and Controls
+---
```

Comment:
Shouldn't need a sidebar label value here since the page title is the same and will dictate the sidebar label.

Reply:
Makes sense.

Reply:
Addressed om [`9838d09` (#5425)](https://github.com/cypress-io/cypress-documentation/pull/5425/commits/9838d0989532a0989f7e44c2bdd9b3346fc37ac4)